obsidian.css

```diff
@@ -7976,13 +8009,13 @@ body .block-ref-count:hover {
 
 
 /*Hide Header Line*/
-.h-line h1:after,
-.h-line h2:after,
-.h-line h3:after,
-.h-line h4:after,
-.h-line h5:after,
-.h-line h6:after,
-.h-line .cm-s-obsidian pre.HyperMD-header:after {
+.sphd-und-1 h1:after,
```

Comment:
The class is perfect, I think we should keep the `.h-line` class for those that might want to toggle it off with 1 toggle.

So something like this should also allow for the style settings to still toggle them off individually:
```css
.sphd-und-1 h1:after,
.sphd-und-2 h2:after,
.sphd-und-3 h3:after,
.sphd-und-4 h4:after,
.sphd-und-5 h5:after,
.sphd-und-6 h6:after,
.h-line h1:after,
.h-line h2:after,
.h-line h3:after,
.h-line h4:after,
.h-line h5:after,
.h-line h6:after,
.h-line .cm-s-obsidian pre.HyperMD-header:after
```

Reply:
Reinstated. Since this version of it causes the h-line to trump the individual settings, I added a note to the description for the group of individual settings - just so users would know why if they're not seeing their changes take effect there.